sa/proto/sa.proto

```diff
@@ -447,3 +447,8 @@ message RateLimitOverrideResponse {
   bool enabled  = 2;
   google.protobuf.Timestamp updatedAt = 3;
 }
+
+// Used internally for storage in the DB, not for RPCs.
+message Authzs {
+  repeated int64 authzIDs = 1;
+}
```

Comment:
Personally I'd break this out into a separate file, but maybe that plays havoc with our generate.sh.